src/team_data.rs

```diff
@@ -4,7 +4,8 @@ use rust_team_data::v1::{Teams, ZulipMapping, BASE_URL};
 use serde::de::DeserializeOwned;
 
 async fn by_url<T: DeserializeOwned>(client: &GithubClient, path: &str) -> anyhow::Result<T> {
-    let url = format!("{}{}", BASE_URL, path);
+    let base = std::env::var("TEAMS_API_URL").unwrap_or(BASE_URL.to_string());
```

Comment:
mention `TEAMS_API_URL` in the README with purpose and a suggested default value?

Reply:
Yes, documenting for developers the env vars needed to run the triagebot (new one: `TEAMS_API_URL`). I only half agree but probably just unimportant nits, I guess :-)